Gruntfile.js

```diff
@@ -161,15 +171,17 @@ module.exports = function (grunt) {
 	grunt.registerTask('jenkins-extensive', ['exec:coverageSingleExtensive']);
 	grunt.registerTask('coverageReport', ['exec:coverageReport']);
 	grunt.registerTask('eslint-nofix', ['eslint']);
-	grunt.registerTask('test', ['eslint', 'exec:coverage']);
```

Comment:
Missing most important task: `test`. Also `test-functional`. Please recover them.

Reply:
`test` script was/is not working anymore. `test-functional` either. They are split it into several scripts.

Reply:
How is that? Just chaining grunt task should work just fine:
```
	grunt.registerTask('test-functional', ['test-functional-ws', 'test-functional-http']);
        grunt.registerTask('test', ['test-functional', 'test-unit']);
```